src/ripple/app/misc/AmendmentTableImpl.cpp

```diff
@@ -100,8 +115,78 @@ class AmendmentTableImpl : public AmendmentTable
 void
 AmendmentTableImpl::addInitial ()
 {
-    // For each amendment this version supports, construct the AmendmentState object by calling
-    // addKnown. Set any vetoes or defaults. A pointer to the AmendmentState can be stashed
+    std::set<std::pair<uint256, std::string>> toAdd;
+
+    for (auto const& _ : detail::PreEnabledAmendments)
```

Comment:
While I'm not in favor of using `_` as an identifier, I vaguely remember that this is one of Howard's habits.  I believe a lone underscore squeaks by the rules because it is a leading underscore that is not followed by anything.  @HowardHinnant, am I mis-remembering?  That's not to say that I think a lone `_` is necessarily a *good* name, but it's possibly legal.